test/lib/omniauth/strategies/openid_connect_test.rb

```diff
@@ -524,7 +524,7 @@ def test_callback_phase_with_jwks_uri
                        .returns(
                          Faraday.new do |builder|
                            builder.adapter :test do |stubs|
-                             stubs.get(strategy.options.client_options.jwks_uri) { [200, {}, jwks.to_json] }
+                             stubs.get(strategy.options.client_options.jwks_uri) { [200, {}, JSON.parse(JSON.generate(jwks))] }
```

Comment:
`jwks` has Symbol as value, so use `JSON.parse(JSON.generate(jwks))` to convert into String.

Reply:
#157 cleans up this stubbing.